ui-library-filesharing-ui-components/app/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
every time we say Create React App can we use something like UI Library File Sharing UI Component Sample? I admit this is a little wordy but at least its precise about what we are doing?